plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -164,8 +172,18 @@ public class OracleClient
 
     private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd");
     private static final DateTimeFormatter TIMESTAMP_SECONDS_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss");
-    private static final DateTimeFormatter TIMESTAMP_MILLIS_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss.SSS");
 
+    private static final Map<Integer, DateTimeFormatter> DATE_FORMATTER_MAP = Map.of(
```

Comment:
nit: I would use `ImmutableMap.<Integer, DateTimeFormatter>builder()` considering future change. `Map.of` doesn't accept arguments more than 10 entries.